tVip.sp

```diff
@@ -644,4 +656,60 @@ stock bool isVipCheck(int client) {
 public void SQLErrorCheckCallback(Handle owner, Handle hndl, const char[] error, any data) {
 	if (!StrEqual(error, ""))
 		LogError(error);
-} 
+}
+
+
+
+//Natives
+
+public int NativeGrantVip(Handle myplugin, int argc)

```

Comment:
It would be better to allow the Console as Admin since what this method archieved can be done with a command too